couler/tests/test_data/output_golden_1.yaml

```diff
@@ -51,3 +48,6 @@ spec:
           - cowsay
         args:
           - "{{inputs.parameters.para-consumer-0}}"
+  volumes:
```

Comment:
This change doesn't look correct? It should be part of the template? (similar in other YAML files)